cocos/tween/actions/action-instant.ts

```diff
@@ -265,7 +265,7 @@ export class CallFunc extends ActionInstant {
      * @param {*|Null} [data] data for function, it accepts all data types.
      * @return {Boolean}
      */
-    initWithFunction (selector: any, selectorTarget?: any, data?: any): boolean {
+    initWithFunction<T> (selector?: Function | null, selectorTarget?: T, data?: any): boolean {
```

Comment:
Why use `T` here?

Reply:
Since selectorTarget could be any user types, we use template type rather than `any` to make the type more strict.